_posts/2019-12-20-this-blog-post-contains-important-information-for-some-percentage-of-its-readers.markdown

```diff
@@ -0,0 +1,150 @@
+---
+layout: epic
+title: This Blog Post Contains Important Information For Some Percentage of its Readers
```

Comment:
I disagree with the feedback you got about the title, I think the original "Is this you?" made a lot of sense. Not all blog posts need super SEO-y titles ([example 1](https://artsy.github.io/blog/2018/01/24/kubernetes-and-hokusai/), [example 2](https://artsy.github.io/blog/2015/08/27/swift-trek-2-the-wrath-of-count/)).

I'd recommend something like:

```suggestion
title: "Is this you? No, really, is this you?"
```

Or something like that. The question grabs the reader's interest, and the follow-up reinforces that this post might be for them.

Reply:
I like this and will take this suggestion, thanks!